handler/handler.go

```diff
@@ -15,6 +15,8 @@ func SearchStudentMaterial(resp http.ResponseWriter, req *http.Request) {
 	keys := req.URL.Query()
 	resp.Header().Add("Content-Type", "application/json")
 	var grade uint32
+	var language string
+	language = "ENGLISH"
```

Comment:
We use small letters now "en", "hi" and so on. Make appropriate change to CSV as well. We won't be typing out the full language.

Also why not ```language := "en"```

Reply:
Since it has the function scope, it should be fine right?